stage-migrid.yml

```diff
@@ -13,3 +13,4 @@
     - copy_static_files
     - update_status_html
     - promtail_migrid
+    - migrid_logstash_gdp
```

Comment:
If the whole role should only be applied if the server is a SIF server, the `when` could also be placed here in the playbook instead of the main.yml in the role.

Eg.

```
roles:
    - { role: migrid_logstash_gdp, when: migrid_base_type == "sif" }
```